src/calendar/day/basic/index.js

```diff
@@ -69,6 +71,10 @@ export default class BasicDay extends Component {
     return disableTouch;
   }
 
+  isSelected() {
```

Comment:
See my comment in the Week component. 
Adding another source for the "selected" mode can lead to bugs in the future.

Reply:
I agree in general, but I do think we need to separate user vs internal changes/update since the basic components are controlled while the more complex, like expandableCalendar, are not.